src/hotspot/cpu/aarch64/stubGenerator_aarch64.cpp

```diff
@@ -1519,9 +1519,9 @@ class StubGenerator: public StubCodeGenerator {
       __ push(RegSet::of(d, count), sp);
     }
     {
-      // UnsafeCopyMemory page error: continue after ucm
+      // UnsafeMemoryAccess page error: continue after ucm
       bool add_entry = !is_oop && (!aligned || sizeof(jlong) == size);
-      UnsafeCopyMemoryMark ucmm(this, add_entry, true);
+      UnsafeMemoryAccessMark ucmm(this, add_entry, true);
```

Comment:
May be rename `ucmm` and other related locals too to avoid confusion. Word `ucm` in comments too.

Reply:
Done.  Comment says `unsafe access` instead of `ucm` and `umam` instead of `ucmm`.